implementations/micrometer-registry-wavefront/src/test/java/io/micrometer/wavefront/WavefrontHistogramTest.java

```diff
@@ -0,0 +1,4 @@
+package io.micrometer.wavefront;
+
+public class WavefrontHistogramTest {
```

Comment:
Em, I feel I have more comments to this class than lines inside :) 
* copyrights
* as per JUnit 5 conventions class should be package-private
* very minor: where is content ? :)

Reply:
Bad commit, filling in the gaps now.

Reply:
Updated the access level.